s3/sc-s3-encrypted-ra.yaml

```diff
@@ -51,6 +51,14 @@ Resources:
         ServerSideEncryptionConfiguration:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: AES256
+      LifecycleConfiguration:
+        Rules:
+          - Id: IntelligentTieringRule
+            Prefix: IntTier
```

Comment:
The Prefix is used to identifies which objects the rule applies to, in particular it would be an object key prefix so I do not think we need this.

Reply:
removed